CSS/utility.css

```diff
@@ -0,0 +1,279 @@
+@import url('https://fonts.googleapis.com/css2?family=Ubuntu&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@200;400&family=Ubuntu&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Kalam:wght@300&family=Roboto:wght@100&display=swap');
+body {
+    font-family: "Raleway", "Ubuntu", "HelveticaNeue", "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+
+/*---------------------------Padding-----------------------------------*/
+
+.padding-xxsmall {
+    padding: 0.25rem;
+}
+
+.padding-xsmall {
+    padding: 0.5rem;
+}
+
+.padding-small {
+    padding: 1rem;
+}
+
+.padding-medium {
+    padding: 1.5rem;
+}
+
+.padding-large {
+    padding: 2rem;
+}
+
+.padding-xlarge {
+    padding: 2.5rem;
+}
+
+.padding-xxlarge {
+    padding: 3rem;
+}
+
+
+/*--------------------------------------------*/
+
+.padding-top-bottom-small {
+    padding: 0.25rem 0rem 0.25rem 0rem;
+}
+
+.padding-top-bottom-medium {
+    padding: 0.5rem 0rem 0.5rem 0rem;
+}
+
+.padding-top-bottom-large {
+    padding: 1rem 0rem 1rem 0rem;
+}
+
+
+/*---------------------------------------------*/
+
+.padding-left-right-xsmall {
+    padding: 0rem 0.25rem 0rem 0.25rem;
+}
+
+.padding-left-right-small {
```

Comment:
rather than naming padding left and right , write padding horizontal or padding vertical respectively